lib/vdsm/storage/blockVolume.py

```diff
@@ -421,8 +421,8 @@ def optimal_size(self):
                 "irs", "volume_utilization_chunk_mb") * MiB
             optimal_size = actual_size + free_space
         else:
-            # For internal volumes, extend to minimal volume size.
-            optimal_size = max(actual_size, sc.MIN_CHUNK)
+            # For internal volumes, use the actual size.
+            optimal_size = actual_size
```

Comment:
Need to ensure size >= self.align_size here.

Reply:
The next line rounds optimal size to align size, and optimal size cannot be zero since actual_size includes qcow2 metadata size.